windows/src/global/delphi/general/Keyman.System.Settings.pas

```diff
@@ -409,6 +409,20 @@   TKeymanSettings = class(TObjectList<TKeymanSetting>)
       ValueType: kstInteger
     ),
 
+    // kmshell:Keyman.System.KeymanStartTask
+    (
+      ID: 'engine.compatibility.use_autostart_task';
```

Comment:
Is this tied to the Keyman configuration option "Start when Windows starts"?

Reply:
No, it's unrelated. This autostart only happens when the user selects a Keyman keyboard using the Windows language selector, and Keyman is not already running.